src/charm.py

```diff
@@ -61,7 +64,9 @@ def _hydra_layer(self) -> Layer:
                     "override": "replace",
                     "summary": "entrypoint of the hydra-operator image",
                     "command": f"hydra serve all --config {self._hydra_config_path} --dev",
-                    "startup": "enabled",
+                    "startup": "disabled"
```

Comment:
Why add the pebble layer on pebble ready when the database is not yet created. We could simply add the pebble on database created instead.

Reply:
Thanks for your review @gruyaume.
I think the layer should be added on both `database_created` and `pebble_ready`. The reason is, a new database is created only once on install, but in case the container crashes only `pebble_ready` will be re-run, so the layer needs to be added here again.
In other reviews we also agreed to add a layer even if the database is not created. What do you think?